robot_smach_states/src/robot_smach_states/navigation/follow_operator.py

```diff
@@ -175,7 +175,7 @@ def _register_operator(self):
         """
         start_time = rospy.Time.now()
 
-        self._robot.head.look_at_standing_person(distance=1.5)
+        self._robot.head.look_at_standing_person(distance=1.5, height=1.3)
```

Comment:
too many magic numbers. This would be alright in a challenge but in a smach state its going to give us headaches later on.

Reply:
The question is how to split the functionality...